gcode.c

```diff
@@ -2436,7 +2445,18 @@ status_code_t gc_execute_block (char *block)
                     gc_block.values.xyz[idx] = gc_state.g92_coord_offset[idx];
             } while(idx);
             break;
-
+            
+#if ENABLE_ACCELERATION_PROFILES
+        case NonModal_SetAccelerationProfile:
+            if (!gc_block.values.p){
```

Comment:
This should be !gc_block.words.p to check if a value has been supplied.
IMO also check gc_block.words.e, if supplied (true) error out since it is not supported?